### PR TITLE
Expose the ioredis .quit() function

### PR DESCRIPTION
**Pull request by adamrights**

### CODE REVIEW

**Comment by adamrights** [comment]
Need to check version of ioredis to see if quit exists. Might be disconnect at our version: https://github.com/luin/ioredis/blob/master/API.md#redisdisconnect

**Comment by adamrights** [comment]
Yes. Will write a test and actually run code today (this PR was a placeholder, but we've added pm2 clustering across the node web apps using distribucache). 

Although, I need to double check the api as it might only expose .disconnect() at the current ioredis version.

File: test/util-test.js [hunk 1]
```diff
@@ -103,6 +103,11 @@ describe('datastore/redis/util', () => {
       client.config.d.should.eql(2);
     });
 
+    // describe('with ability to quit', () => {
```
Comment (nemtsov) [inline test/util-test.js]: remove the dead code, plz

File: test/RedisStore-test.js [hunk 2]
```diff
@@ -50,6 +51,15 @@ describe('RedisStore', () => {
     util.ensureKeyspaceNotifications.calledOnce.should.be.ok();
   });
 
+  it('should quit', done=>{
+    redisClient.quit.withArgs().yields(null, 'ok');
+    unit.quit(function(err){
```
Comment (nemtsov) [inline test/RedisStore-test.js]: nit: `function (err) {`

File: test/RedisStore-test.js [hunk 3]
```diff
@@ -50,6 +51,15 @@ describe('RedisStore', () => {
     util.ensureKeyspaceNotifications.calledOnce.should.be.ok();
   });
 
+  it('should quit', done=>{
```
Comment (nemtsov) [inline test/RedisStore-test.js]: nit: `done => {` we should really just bring in [Prettier](https://github.com/prettier/prettier) and never have to talk about spacing again.

File: test/util-test.js (hunk 1 above)
Comment (adamrights) [reply test/util-test.js]: certainly

File: test/RedisStore-test.js (hunk 3 above)
Comment (adamrights) [reply test/RedisStore-test.js]: I agree. I like how the trend has moved toward automating such.

**Comment by adamrights** [comment]
I think we're good to merge?

**Comment by nemtsov** [comment]
> I think we're good to merge?

Do it!

**Pull request merged**